src/translations/pl/translation.json

```diff
@@ -444,46 +444,46 @@
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}": "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}}",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_many": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_other": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds",
-    "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}": "While target has the {{effectNames, list}} effect(s) on their {{bodyParts, list(type: disjunction)}}",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_one": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
+    "While having the {{effectNames, list}} effect(s) on your {{bodyParts, list(type: disjunction)}} for {{operator}} {{count}} seconds_few": "Mając {{effectNames, list}} efekt(-y) na twojej {{bodyParts, list(type: disjunction)}} na {{operator}} {{count}} sekund",
```

Comment:
The translation of the seconds for `seconds_few` shouldn't be `sekundy`?

Eg:

> While having the **pain** effect(s) on your **arm** for **3** seconds

p.s. this comment applies to other strings too where you see the `seconds_few` quantifier